server.go

```diff
@@ -272,24 +281,8 @@ func (s *Server) handleRequest(w http.ResponseWriter, r *http.Request, route *Ro
 	req.Header.Set(HeaderViewProxyOriginalPath, r.URL.RequestURI())
 	results, err := req.Do(ctx)
 
-	if err != nil {
-		if s.OnError != nil {
-			s.OnError(w, r, err)
-			return
-		} else {
-			w.WriteHeader(http.StatusInternalServerError)
-			w.Write([]byte("500 internal server error"))
-			return
-		}
-	}
-
-	resBuilder := newResponseBuilder(*s, w)
-	resBuilder.SetLayout(results[0])
-	resBuilder.SetHeaders(results[0].HeadersWithoutProxyHeaders(), results)
-	resBuilder.SetFragments(results[1:])
-	elapsed := time.Since(startTime)
-	resBuilder.SetDuration(elapsed.Milliseconds())
-	resBuilder.Write()
+	r = r.WithContext(multiplexer.ContextWithResults(r.Context(), results, err, startTime))
```

Comment:
Super minor, and no strong feelings either way, but would it make sense to have `startTime` stored in a separate context value?

Reply:
Yeah, it seemed okay to surface on the new `Results` interface but admittedly I don't have a use for it right now in consumers.

Let me remove it for now from results API to simplify it.